sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -289,6 +213,34 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
     Azure::Response<PurgedSecret> PurgeDeletedSecret(
         std::string const& name,
         Azure::Core::Context const& context = Azure::Core::Context()) const;
-  };
 
+    /**
+     * @brief Delete a secret from a specified key vault.
+     *
+     * @remark The DELETE operation applies to any secret stored in Azure Key Vault.
+     * DELETE cannot be applied to an individual version of a secret.
+     * This operation requires the secrets/delete permission.
+     *
+     * @param name The name of the secret<span class="x x-first x-last">.</span>
+     * @param context The context for the operation can be used for request cancellation.
+     */
+    Azure::Security::KeyVault::Secrets::KeyVaultDeleteSecretOperation StartDeleteSecret(
+        std::string const& name,
+        Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Recover the deleted secret to the latest version.
+     *
+     * @remark Recovers the deleted secret in the specified vault.
+     * This operation can only be performed on a soft-delete enabled vault.
+     * This operation requires the secrets/recover permission.
+     *
+     * @param name The name of the secret<span class="x x-first x-last">.</span>
```

Comment:
Are you adding this html tag `span class=...`  for the dot?  I have seen it before.   I wonder what's this is about?

Reply:
it came from a comment in a PR a long time ago, so it just got perpetuated through copy paste i don't think it's of any use. I removed it for all occurences.